docs/tools/cli/service/privatelink.rst

```diff
@@ -0,0 +1,423 @@
+``avn service privatelink``
+==============================================
+
+Here you’ll find the full list of commands for ``avn service privatelink``.
+
+
+Manage Aiven PrivateLink service for AWS and Azure
+--------------------------------------------------------
+
+.. _avn_service_privatelink_availability:
+
+``avn service privatelink availability``
+'''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''
+
+Lists PrivateLink cloud availability and prices.
+
+.. _avn_service_privatelink_aws_connection_list:
+
+``avn service privatelink aws connection list``
+'''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''
+
+Lists PrivateLink connection information for a service.
+
+.. list-table::
+  :header-rows: 1
+  :align: left
+
+  * - Parameter
+    - Information
+  * - ``service_name``
+    - The name of the service
+  * - ``--project``
+    - The project to fetch details for
+
+**Example:** List PrivateLink for ``kafka-12a3b4c5`` service.
+
+::
+
+    avn service privatelink aws connection list kafka-12a3b4c5
+
+An example of output:
+
+.. code:: text
+
+    {
+      "dns_name": "vpce-0123456789abc1345-qfhrjbis.vpce-svc-0abcdef0123456789.us-east-1.vpce.amazonaws.com",
```

Comment:
Minor tip: we could shorten the `dns_name` so people don't have the scrolling at the bottom

Reply:
You are correct, We can leave it as it is if it makes sense to show the format